app/src/main/res/layout/activity_levels_view.xml

```diff
@@ -6,179 +6,209 @@
     android:layout_height="match_parent"
```

Comment:
Good job with the changes. Layout looks good now both on my phone and Tablet ;) I'll suggest my approach to it on the branch: xyz